src/pretix/control/templates/pretixcontrol/organizers/customer.html

```diff
@@ -71,12 +72,22 @@ <h3 class="panel-title">
                             <dt>{% trans "Notes" %}</dt>
                             <dd>{{ customer.notes|linebreaks }}</dd>
                         {% endif %}
-                        <dt>{% trans "Lifetime spending" %}</dt>
-                        {% for s in lifetime_spending %}
-                            <dd>{{ s.spending|money:s.currency }}</dd>
-                        {% endfor %}
                         </dl>
                     </form>
+                    {% if lifetime_spending %}
+                        <div class="text-right col-lg-4 col-md-6 col-sm-12"
+                                data-toggle="tooltip"
+                                title="{% trans "This includes all paid orders by this customer across all your events." %}"
+                        >
+                            <p class="text-muted">{% trans "Lifetime spending" %}</p>
+                                <h3 class="text-success">
+                                    {% for s in lifetime_spending %}
+                                            {{ s.spending|money:s.currency }}
+                                    {% endfor %}
+                                </h3>
```

Comment:
`<h3>` has semantic meaning which is not fitting here, for a bigger font-size better use bootstraps `.h3` css-class. Probably semantically the most correct way – and aligned with the other infos marked up as `<dl><dt>` – would be to use a `<dl>`. So:

```suggestion
                            <dl class="horizontal">
                                <dt class="text-muted">{% trans "Lifetime spending" %}</dt>
                                <dd class="text-success h3">
                                    {% for s in lifetime_spending %}
                                            {{ s.spending|money:s.currency }}
                                    {% endfor %}
                                </dd>
                            </dl>
```

You could even go further (better!) and put everything inside the already existing `<dl>` and use `<dl><div class="col …"><dt>…</dt><dd>…</dd></div><div class="col …"><dt>…</dt><dd>…</dd></div></dl>` to group them into columns – `<div>`s are allowed in `<dl>`. That way the `<form>` can hold the obligatory `.row` css class for columns to correctly work (contain the float based columns).

Reply:
I tried the version with `<div>`s, but in the end, I went with a separate `<dl>` with no `dl-horizontal` class to ensure no weird formatting occured.